### DISCUSSION TITLE
background colour

### DISCUSSION THREAD

**Discussion by AlKoriks** [sumatrapdfreader/sumatrapdf#2508] [General]
Hi evrbdy! Many thanks to Mr Kowalczyk for Sumatra project, it helps me a lot! 
A small note. Is it possible to mark the active document tab name (in the tab bar or tab line above the command line) or the whole tab any way? For examlple, by changing colour or using an extra bold name font, or blinking, or smth else? Because the tab background of nonactive docs is light-grey, and the active tab is white, it's rather hard to choose the active tab quickly, especially when several docs are opened. Thanks for response.

**Comment by GitHubRulesOK** [comment]
only way is to recode the colours in source (though I have done changes by poking/modding the binary as proof of concept but naturally the address changes on every compile)
The core color for tabs is part of Windows dressing and can be changed by using skinning tools like [WindowBlinds](https://www.stardock.com/products/windowblinds/) see https://github.com/sumatrapdfreader/sumatrapdf/discussions/2441
